activities/tool-management/odoo.md

```diff
@@ -0,0 +1,20 @@
+---
+type: resource
+---
+
+# Odoo Community
+
+Odoo is a suite of open source business apps that cover all your company needs: CRM, eCommerce, accounting, inventory, point of sale, and project management.
+
+## Guidance on using Odoo
+
+We use Odoo's CRM and project management capabilities for managing our Codebases and Membership pipelines.
+
+## Odoo Management
+
+Odoo Community is currently hosted and maintained by a service provider in the UK. We are migrating our instance to a Dutch provider late November 2020. The Operations coordinator takes care of user administration, reports bugs and is the contact person for the service provider.
+
+## Odoo Privacy Policy
+
+> Our future service provider has published their 'verwerkersovereenkomst' [on their website](https://www.openworx.nl/verwerkersovereenkomst).
+Their [privacy policy](https://www.openworx.nl/privacy-statement) (also in Dutch) is available as well.
```

Comment:
This is inconsistent, do we work with OpenWorx yet?

Reply:
cc @demkodo!

Reply:
Hey guys, Opus VL never got back to me re; GDPR, they only had GDPR regulations in place for is using their website, not their hosted services, which is truly abhorrent and another reason I'm glad we are leaving. Indeed, openworx isn't active yet but will be soon, hence "our future service provider". So, a middle road will be not having anything around Odoo and GDPR until the migration is complete. Lemme know.